packages/cli/src/config/settings.ts

```diff
@@ -27,6 +28,56 @@ export const USER_SETTINGS_PATH = Storage.getGlobalSettingsPath();
 export const USER_SETTINGS_DIR = path.dirname(USER_SETTINGS_PATH);
 export const DEFAULT_EXCLUDED_ENV_VARS = ['DEBUG', 'DEBUG_MODE'];
 
+// As defined in spec.md
+const MIGRATION_MAP: Record<string, string> = {
```

Comment:
nice! please also add tests that verify that this is used to successfully lto migrate legacy schemas. In particular I'd be worried about making sure that the merging of legacy array like properties is handled correctly.

Reply:
Done.